directories in .hak/hakModules are already symlinked inside node_modules,
and as such are already being copied by default. this makes tasje fail with:
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value:
FileAlreadyWritten("/node_modules/keytar/package.json")', src/main.rs:200:18
```

console.log interferes with tasje, which reads config from node stdout

--- ./electron-builder.ts.orig
+++ ./electron-builder.ts
@@ -72,10 +72,6 @@
     console.warn(`No VARIANT_PATH specified, using default variant configuration '${DEFAULT_VARIANT}':`);
 }
 
-for (const key in variant) {
-    console.log(`${key}: ${variant[key]}`);
-}
-
 interface Configuration extends BaseConfiguration {
     extraMetadata: Partial<Pick<Pkg, "version">> & ExtraMetadata;
     linux: BaseConfiguration["linux"];
@@ -112,10 +108,6 @@
     },
     files: [
         "package.json",
-        {
-            from: ".hak/hakModules",
-            to: "node_modules",
-        },
         "lib/**",
     ],
     extraResources: ["build/icon.*", "webapp.asar"],
